### PR TITLE
Add frequency to alert messages

### PR DESCRIPTION
**Pull request by CoryDuncan**

### CODE REVIEW

**Comment by CoryDuncan** [comment]
@sethherr 

> Did you remove "1" from the frequency because of its redundancy, spacing issues or something else?

Only for readability

**Comment by troy** [comment]
lgtm. I could imagine `frequency_phrase()` returning `in the past X` since that's used so often.

File: services/mail.rb [hunk 1]
```diff
@@ -111,7 +112,7 @@ def html_email
   def text_email
     erb(unindent(<<-EOF), binding)
 
-      <%= Pluralize.new('event', :count => event_count) %> matched your "<%= payload[:saved_search][:name] %>" search.
+      <%= Pluralize.new('event', :count => event_count) %> matched your "<%= payload[:saved_search][:name] %>" search in the past <%= translate_frequency(payload[:frequency]) %>..
```
Comment (armchairlinguist) [inline services/mail.rb]: Is there a double `..` here now?

Comment (CoryDuncan) [inline services/mail.rb]: @armchairlinguist Good catch, fixed now

**Comment by CoryDuncan** [comment]
@troy updated in https://github.com/papertrail/papertrail-services/commit/530e944b8c9e9cc601ec7a703f33a3fa9d0c5bc0

**Pull request merged**